app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,16 @@
+from .dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
```

Comment:
Fix the indentation here, and move the nickname to the new line. The same in all other similar cases